src/domain/card/components/CardCheap/CardCheap.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+interface CardProps {
+  scale: number;
```

Comment:
imo
scaleじゃなくてclassName?:stringで受け取れるようにしてください

cn("既存のCSS",className)
```suggestion
  className?: string;
```

className={cn("既存のクラス",className)}

Reply:
動的にscaleしようと思ったらclassNameベースだと無理じゃない？

Reply:
tailwindならjs使った動的なscaleじゃない限りはコンパイル時に値として入るはず。

いずれにしてもこのコンポーネントの責務はカードのサイズを変えることじゃなくてカードを表示することであるからscaleに関しては親で指定したい

Reply:
tailwindならjs使った動的なscaleじゃない限りはコンパイル時に値として入るはず。

いずれにしてもこのコンポーネントの責務はカードのサイズを変えることじゃなくてカードを表示することであるからscaleに関しては親で指定したい